Model/app.js

```diff
@@ -13,17 +14,10 @@ app.controller('myController', ['$scope', ($scope) => {
   $scope.filesToSearch = [];
   $scope.fileContent = {};
   $scope.search = {};
-
-  const file = document.getElementById('files');
-  const indexBtn = document.getElementById('index');
-  const selectFiles = document.getElementById('selectFiles');
-  const search = document.getElementById('searchTerms');
-  const fileName = $scope.selectFiles;
-
-  file.addEventListener('change', () => {
-    $scope.uploadFile(file);
+  const fileUpload = document.getElementById('files');
+  fileUpload.addEventListener('change', () => {
+    $scope.uploadFile(fileUpload);
   });
-
   $scope.uploadFile = (file) => {
     file = file.files;
     for (let i = 0; i < file.length; i += 1) {
```

Comment:
You can use a `forEach` on the `file` array so that the `eslint-disable` would not be necessary. 
```
file.forEach(selectedFile => {

);
```

You can use the `selectedFile` variable in place of `fie[i]`